### PR TITLE
Use "equality operator" instead of "identity operator" in JS basics

### LINKED ISSUE #16222: "JavaScript basics" content - Conditionals - uses "identity operator" instead of "equality operator"
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics

### What specific section or headline is this issue about?

Conditionals

### What information was incorrect, unhelpful, or incomplete?

Please see [https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics#conditionals](url)

> ... This uses the **identity** operator ... 

### What did you expect to see?

> ... This uses the **equality** operator ...

### Do you have any supporting links, references, or citations?

Please see the table row where "Operator" is "Equality" in the table in this section:
[https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics#operators](url)

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/getting_started_with_the_web/javascript_basics`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/JavaScript_basics
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md
* Last commit: https://github.com/mdn/content/commit/30fd1df757e622f4067660e5d731b0f3b83936aa
* Document last modified: 2022-05-09T04:27:46.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Motivation

Fix #16222

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ if(iceCream === 'chocolate') {
 }
 ```
 
-The expression inside the `if( ... )` is the test. This uses the identity operator (as described above) to compare the variable `iceCream` with the string `chocolate` to see if the two are equal. If this comparison returns `true`, the first block of code runs. If the comparison is not true, the second block of code—after the `else` statement—runs instead.
+The expression inside the `if( ... )` is the test. This uses the equality operator (as described above) to compare the variable `iceCream` with the string `chocolate` to see if the two are equal. If this comparison returns `true`, the first block of code runs. If the comparison is not true, the second block of code—after the `else` statement—runs instead.
```
Comment (OnkarRuikar) [inline files/en-us/learn/getting_started_with_the_web/javascript_basics/index.md:290]:
```suggestion
The expression inside the `if( ... )` is the test. This uses the strict equality operator (as described above) to compare the variable `iceCream` with the string `chocolate` to see if the two are equal. If this comparison returns `true`, the first block of code runs. If the comparison is not true, the second block of code—after the `else` statement—runs instead.
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

**Comment by teoli2003** [comment]
We also should use _strict equality_ at line 232.

**Review by teoli2003 — APPROVED** [review]
Thank you!

**Comment by SeaView1** [comment]
Thank you  @Josh-Cena , @OnkarRuikar  and  @teoli2003  for your help with this, including correction of my correction to “strict” equality!

**Pull request merged**